src/murfey/server/api/clem.py

```diff
@@ -687,3 +694,72 @@ def process_raw_tiffs(
         messenger=_transport_object,
     )
     return True
+
+
+class AlignAndMergeParams(BaseModel):
+    # Processing parameters
+    series_name: str
+    images: list[Path]
+    metadata: Path
+    # Optional processing parameters
+    crop_to_n_frames: Optional[int] = None
```

Comment:
As noted on the cryoemservices PR, these could possibly default to something else rather than None